### PR TITLE
Add missing SDKs

### PR DESCRIPTION
**Pull request by mlorbetske**
@rohit21agrawal @piotrpMSFT @vijayrkn @srivatsn 

NuGet.BuildTasks.Pack 4.0.0-rc2
Microsoft.NET.Sdk.Publish (parameterized version)
Microsoft.NET.Sdk.Web.ProjectSystem (parameterized version)

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build please.

**Comment by piotrpMSFT** [comment]
@MattGertz

File: build/Microsoft.DotNet.Cli.BundledSdks.props [hunk 1]
```diff
@@ -3,7 +3,10 @@
   
   <ItemGroup>
     <!-- CLI cannot use the latest SDK until we move away from SDK PackageRef -->
+    <BundledSdk Include="NuGet.Build.Tasks.Pack" Version="4.0.0-rc2" />
     <BundledSdk Include="Microsoft.Net.Sdk" Version="1.0.0-alpha-20161202-1" />
```
Comment (vijayrkn) [inline build/Microsoft.DotNet.Cli.BundledSdks.props]: Microsoft.Net.Sdk version should be 20161203-1 to match the version of other packages.

Comment (mlorbetske) [inline build/Microsoft.DotNet.Cli.BundledSdks.props]: Updated

**Comment by MattGertz** [comment]
Approved.

**Pull request merged**